scripts/area_plot.py

```diff
@@ -0,0 +1,676 @@
+#!/bin/env python3
+
+from sys import argv, exit
+import argparse
+import warnings
+
+from matplotlib import rcParams
+from matplotlib import cm
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+
+"""
+Generates a pretty areaplot from a collapsed feature table.
+"""
+__author__ = "Justine Debelius"
+__date__ = "2020"
+__version__ = "1.0"
+
+__author__: 'JW Debelius'
```

Comment:
These three lines look like won't do much? I don't think it'll produce a syntax error, but it probably won't work either.

Didn't know this was valid syntax. :D